samples/xml-config/pom.xml

```diff
@@ -182,8 +182,9 @@
               <resources>
                 <!-- Labels that are applied to all created objects -->
                 <labels>
-                  <group>quickstarts</group>
-                </labels>
+                    <all>
+                        <group>quickstarts</group>
+                    </all>
```

Comment:
add `</label>` to close labels field here

Reply:
deployments is one of the labels in metadataconfig. On the contrary there is no field deployment in ResourceConfig hence there build fails. And I have tested the build, now it passes.

Reply:
oh, okay. Have you fixed this in external profile as well?

Reply:
Yes everything is correct there, adding the digest again!

Reply:
Fixed, now i guess it is good to merge @rohanKanojia